src/wallet/rpcwallet.cpp

```diff
@@ -2575,6 +2577,267 @@ UniValue fundrawtransaction(const JSONRPCRequest& request)
     return result;
 }
 
+UniValue bumpfee(const JSONRPCRequest& request)
+{
+    if (!EnsureWalletIsAvailable(request.fHelp))
+        return NullUniValue;
+
+    if (request.fHelp || request.params.size() < 2 || request.params.size() > 3)
+        throw runtime_error(
+                            "bumpfee \"txid\" \"output\" \n"
+                            "\nBumps the fee of a opt-in-RBF transaction.\n"
+                            "This command requires that the transaction with the given txid is in the wallet.\n"
+                            "User specifies which output (e.g., a change output) will be used to bump the fee.\n"
+                            "This command will NOT add new inputs.\n"
+                            "Fee must be high enough to pay a new relay fee.\n"
+                            "If tx has child transactions in mempool, the new fee must pay for them as well.\n"
+                            "This command will fail if fee is not high enough or output is not large enough.\n"
+                            "User can specify totalFee, or use RPC setpaytxfee to set a higher fee rate.\n"
+                            "\nArguments:\n"
+                            "1. \"transactionid\"     (string, required) The txid to be bumped\n"
+                            "2. \"output\"            (int, required) The output to be decremented\n"
+                            "3. options               (object, optional)\n"
+                            "   {\n"
+                            "     \"confTarget\"      (int, optional) Confirmation target (in blocks)\n"
+                            "     \"totalFee\"        (numeric, optional) Total fee to pay, in satoshis (not btc)\n"
```

Comment:
I'm not sure if we want absolute fee values here.
The user can't be sure how many inputs are getting added when setting this value, probably resulting in an uncontrollable feerate.

What about switching this to a feerate?